bsp/esp-box-3/esp-box-3.c

```diff
@@ -53,10 +52,13 @@ static const ili9341_lcd_init_cmd_t vendor_specific_init[] = {
     {0, (uint8_t []){0}, 0xff, 0},
 };
 
+#if (BSP_CONFIG_NO_GRAPHIC_LIB == 0)
 static lv_display_t *disp;
 static lv_indev_t *disp_indev = NULL;
 static esp_lcd_touch_handle_t tp;   // LCD touch handle
```

Comment:
`tp` is used in noglib version :(

only `panel_handle` is not

Reply:
You mean used in main button? Sure, I am sorry, I overlooked it.